Marlin/temperature.cpp

```diff
@@ -1949,5 +1956,7 @@ void Temperature::isr() {
     }
   #endif
 
+  cli();
```

Comment:
@Sebastianv650 Is clearing _all interrupts_ here at the end of the temperature ISR actually needed? This won't cause the stepper ISR to cease?